edx_django_utils/monitoring/internal/middleware.py

```diff
@@ -145,6 +154,16 @@ def process_exception(self, request, exception):    # pylint: disable=W0613
         Django middleware handler to process an exception
         """
         self._batch_report()
+        self._tag_root_span(exception)
```

Comment:
Can we add a test, especially so this functionality doesn't get lost by mistake at some point?